test/boost/sstable_resharding_test.cc

```diff
@@ -43,13 +43,13 @@ void run_sstable_resharding_test() {
     auto close_env = defer([&] { env.stop().get(); });
   for (const auto version : writable_sstable_versions) {
     auto s = get_schema();
-    auto cf = env.make_table_for_tests(s);
+    auto cf = env.make_table_for_tests(s, version);
     auto close_cf = deferred_stop(cf);
     std::unordered_map<shard_id, std::vector<mutation>> muts;
     static constexpr auto keys_per_shard = 1000u;
 
     // create sst shared by all shards
-    {
+    sstables::shared_sstable sst = [&] {
```

Comment:
nit, use `std::invoke()` for better readability, so the human reader can understand this is a function call not a definition of a lambda. i learned this from a comment from Avi.

Reply:
will fix in next version